frontend/test/metabase/scenarios/filters/filter-bulk.cy.spec.js

```diff
@@ -390,14 +410,24 @@ describe("scenarios > filters > bulk filtering", () => {
         cy.get("input")
           .eq(0)
           .clear()
-          .type("01/01/2018");
+          .type("01/01/2017");
```

Comment:
I'll take a look.
If this is indeed related to the state management of the `DatePicker` component, I'm afraid that any hack or workaround we implement can break if we update/tweak the component itself.

Given the fact that you've also experienced problems in unit tests, I suspect the component is to blame (and to fix :p)

Reply:
@iethree I can reliably reproduce this failure every single time if I add a 200ms delay to the `type` command, like so: `cy.type("foobar", { delay: 200 });`.

You can try adding `{ delay: 0 }` to speed things up a bit. It will fill out the field in an instant and can potentially eliminate flakiness all together. Seems to be working for me locally - like a charm.